pkgcfg/r_core.pc.acr

```diff
@@ -7,6 +7,6 @@ Name: r_core
 Description: radare foundation libraries
 Version: @VERSION@
 Requires: r_config r_cons r_io r_util r_flag r_asm r_debug r_bin r_lang r_io r_anal r_bp r_egg r_reg r_search r_syscall r_socket r_fs r_magic r_crypto r_arch r_esil
-Libs: -L${libdir} -lr_core  @SSL_LDFLAGS@ @CAPSTONE_LDFLAGS@
+Libs: -L${libdir} -lr_core -lm @SSL_LDFLAGS@ @CAPSTONE_LDFLAGS@
```

Comment:
Probably this -lm must be for the debug

Reply:
Lets try to make this thing work without libm :)

Reply:
ok lets get this thing with -lm for now. ill work on cleaning up the other -lm flags if there's any build issue in the future